### ISSUE TITLE
Opis w przypadku zmiany struktury

### ISSUE THREAD

**Issue by kubaporebski** [kubaporebski/P2#6]
Co się stanie jak się struktura zmieni? Wystarczy opisać co wtedy zrobimy (np. SCD). Nie trzeba przygotowywać mechanizmów.

**Comment by burbongw** [comment]
Nie jestem pewien, czy to dobrze zrozumieliśmy, SCD nie ma wpływu na zmianę struktury. Bardziej chodzi w SCD o logowanie kiedy się zmieniły dane, np. UPDATE.

**Comment by burbongw** [comment]
Co do zmian w strukturze- ciężko cokolwiek wymyślić, to jest API, więc nie mamy za bardzo jak zrobić z tego np. tabeli w bazie źródłowej, ani niczego takiego. dostajemy czyste dane. Musielibyśmy w jakiś sposób tworzyć tabelę dynamicznie, albo napisać skrypt/zrobić coś, co zrobi alter table jeśli definicja tabeli będzie inna niż wynika z pobieranych danych.

**Comment by kubaporebski** [comment]
Prawda, hipotetycznie można jednak założyć że się coś zmieni, jakaś jedna kolumna powiedzmy/parametr. Ale to jutro może obgadamy 🙂

**Comment by kubaporebski** [comment]
Może być taka hipotetyczna sytuacja jak np. podział kolumny Name na LongName i ShortName w Subjects.
